### DISCUSSION TITLE
Any way to "back" button to the source Pagination page?

### DISCUSSION THREAD

**Discussion by bryllyg** [microsoft-search/pnp-modern-search#1411] [Ideas]
### Check existing issues before logging new ones

**Version used**
4.3 and 3.21

**Question**
Is there any way to configure the source on a link so the browser back button takes users back to the specific pagination page they were on? Since the url is not updated, I don't think this is currently doable, but thought I'd ask.

steps:
- User browses results pagination to page 8 of results
- User clicks results link
- User presses browser back button and is now on page 1 of results.
- To continue the search, user has to remember they were on page 8, of the results, and return there manually.

**Screenshots**

**Additional context**
Add any other context which is helpful to understand the question.

**Comment by FranckyC** [comment]
Hi @bryllyg, this is currently not possible as pagination is not handled via query string parameters. I mark it as an enhancement though.

**Comment by wobba** [comment]
I'm very unsure about the usability for this. New results can come in, and visible results per user will/can differ based on ranking and access. So if you link to say page 3, you don't know that page 3 looks the same once you revisit the link. Had a similar discussion internally as well, and I question the real value for the usecase. @bryllyg feel free to add some good reasons where this is useful :)

**Comment by bryllyg** [comment]
@wobba  & @FranckyC  - Thanks! Use case is if the user decides the result they clicked on is not the one they want, and they want to return to the same query and page where they left off to continue searching.  If the back button returns them to tab 1, the onus is on the user to remember which tab they were on before clicking though to a result. In this scenario there's not much chance of the query results changing in the time frame it takes to determine that the first result clicked is not the one the user wants, and returning to the query, and it would ease the burden on the user to not have to remember where they left off.

**Comment by wobba** [comment]
Got it, so it's about state on clicking back in the browser.

**Comment by bryllyg** [comment]
Correct
